build_node/build_node_builder.py

```diff
@@ -198,7 +211,21 @@ def __report_excluded_task(self, task, artifacts):
             'status': 'excluded',
             'artifacts': [artifact.dict() for artifact in artifacts]
         }
-        self.__call_master('build_done', **kwargs)
+        traceback_str = ''
+        try:
+            self.__call_master('build_done', **kwargs)
+        except requests.exceptions.Timeout:
+            traceback_str = traceback.format_exc()
+            self.__retry_counter -= 5
+        except Exception:
+            traceback_str = traceback.format_exc()
+        if traceback_str:
+            self.__logger.error(
```

Comment:
1. logging.exception
2. Don't use f-string or something like this. Use args for logging